types/chai/index.d.ts

```diff
@@ -282,8 +283,12 @@ declare namespace Chai {
 
     interface TypeComparison {
         (type: string, message?: string): Assertion;
+        own: Own;
```

Comment:
I'm not sure these belong here. In your example, you're using `a` as a language chain (which is correct [per the documentation](https://www.chaijs.com/api/bdd/#method_a)), but `a` is not included on `LanguageChains`. I think what you actually want is for `TypeComparison` to extend `Assertion`.

I tested a few scenarios and `a`/`an` can indeed be used as any language chain, so all of the methods on `Assertion` seem to be available on a `TypeComparison`.

Reply:
What I actually want is to have typescript compile properly the examples given in issue #48697. The compiler says that `TypeComparison` is missing some properties so I simply added them to `TypeComparison`.

If I understand correctly, you are wondering if these changes are semantically tied to `TypeComparison` or belong to another class. That is indeed a good remark but I do not understand what you really mean by "all of the methods on `Assertion` seem to be available on a `TypeComparison`" whereas `Assertion` is a strict extension of `TypeComparison `.

Reply:
In order to have `an.own` chain accepted by typescript as it is by javascript, the type of `an` must have a property name `own`. Actually `an` is a `TypeComparison`  so `TypeComparison` must declare `own`.
If you think that `an` should not be a `TypeComparison`, then this is out of the scope of that pull request because it is a major design modification, not just fixing an issue.